src/__tests__/cache.ts

```diff
@@ -0,0 +1,132 @@
+import { Cache } from '../Cache';
+import { TTL_1H } from '../constants';
+
+const items: any = {};
+
+jest.mock('@react-native-async-storage/async-storage', () => ({
```

Comment:
You can add this
```js
import mockAsyncStorage from '@react-native-async-storage/async-storage/jest/async-storage-mock';
```
to a `jest.setup.js` file in root, to auto mock async storage.

Then add

```js
  setupFiles: ['./jest.setup.js'],
```
to `jest.config.js` to load it

Reply:
👌